data/Pyth0nHere.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "Majeed Mujawar",
+  "bio": "DevOps | Networking | Linux | Cloud",
+"links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/sudo_uwu",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/sudo_uwu/",
+      "icon": "FaInstagram"
+    },
+      {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/majeedmujawar01/",
+      "icon": "FaLinkedIn"
```

```suggestion
      "icon": "FaLinkedin"
```

Icons are case sensitive :)